docs/news/news-v4.1.x.rst

```diff
@@ -4,6 +4,31 @@ Open MPI v4.1.x series
 This file contains all the NEWS updates for the Open MPI v4.1.x
 series, in reverse chronological order.
 
+4.1.4 -- May, 2022
+------------------
+:Date: May, 2022
+
+- Fix possible length integer overflow in numerous non-blocking collective
+  operations.
+- Fix segmentation fault in UCX if MPI Tool interface is finalized before
+  ``MPI_Init`` is called.
+- Remove ``/usr/bin/python`` dependency in configure.
+- Fix OMPIO issue with long double etypes.
```

Comment:
etypes?

Reply:
you  can change that to  'elementary types' if you want

Reply:
This is the text that was released in the 4.1.4 NEWS file.